gui/src/main/java/org/verapdf/cli/commands/VeraCliArgParser.java

```diff
@@ -91,9 +94,12 @@
     @Parameter(names = { PROFILES_WIKI_FLAG, PROFILES_WIKI }, description = "The path to the folder containing Validation Profiles wiki.")
     private String profilesWikiPath = "https://github.com/veraPDF/veraPDF-validation-profiles/wiki";
 
-    @Parameter(names = { REPORT_TYPE_FLAG, REPORT_TYPE}, description = "Operation to be performed", converter = ProcessingTypeConverter.class)
+    @Parameter(names = { REPORT_TYPE_FLAG, REPORT_TYPE}, description = "Operation to be performed.", converter = ProcessingTypeConverter.class)
```

Comment:
Strange that it is left here. As I remember, we discussed to left --extract and -f 0 ways to proceed this